apache/ozone
# HDDS-11326. Speed up TestBlockOutputStreamWithFailures

## What changes were proposed in this pull request?

Speed up TestBlockOutputStreamWithFailures. Reduce execution time from 1000s (581 + 418) to 670s Currently:
Tests without flaky 
```
[INFO] Running org.apache.hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures
[INFO] Tests run: 12, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 581.401 s - in org.apache.hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures
```
flaky tests
```
[INFO] Running org.apache.hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures
[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 418.331 s - in org.apache.hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures
```

After changes:
```
[INFO] Running org.apache.hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures
[INFO] Tests run: 20, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 671.981 s - in org.apache.hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures
```
 
Things change in this pr:
1. Remove flaky tag, since it was passed for 1000 iterations without any failures. See https://issues.apache.org/jira/browse/HDDS-6113
2. Setup th cluster only once before all the tests.
3. Restart the datanodes after shutting them down to keep the cluster healthy.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-11326

## How was this patch tested?
CI:
https://github.com/chungen0126/ozone/actions/runs/11561813369
